packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx

```diff
@@ -768,6 +766,195 @@ describe("MafsGraph", () => {
         expect(state.coords).toEqual(expectedCoords);
     });
 
+    it("Quadratic moves points over invalid locations on keystroke", async () => {
```

Comment:
I wanted to write these tests in their respective graph testing files, but I was unable to get the graph to update properly in jest after keyboard interaction. 

Given that `mafs-graphs.test.tsx` was already testing some keyboard functionality, and had the necessary logic for accessing the state directly, I've opted to implement the keyboard tests in this file. 

Happy to take any feedback or suggestions on this as well!